supplementary_materials/notebooks/phi3_rag_on_client.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # ! pip install langchain datasets pandas nltk chromadb sentence-transformers
</pre><p>it is better to use %pip instead of ! pip (! does not guarantee usage the same python executable which used by notebook that may lead to issues on some platform)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2025/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>